### PR TITLE
Add blog

### LINKED ISSUE #1636: Refactor blog

### PR DESCRIPTION
**Pull request by IvanDanilov66**
Fixes #1597 
Fixes #1636 

We are using styled components only on Slider but I think that it should not be a problem, because we have to redo the whole Slider for only that. (The lint problem is fixed)

### CODE REVIEW

**Comment by IvanDanilov66** [comment]
We will be making it without styled components this week, so we will push on this branch. You can merge this now.

File: package.json
Comment (angle943) [inline package.json:75]: why did you downgrade @types/react?

Comment (IvanDanilov66) [reply package.json:75]: Maybe because you updated it in the meantime and I was using the old one.

Comment (IvanDanilov66) [reply package.json]: I have updated it to 17.0.3 (the newest version)

**Review by jamessspanggg — CHANGES_REQUESTED** [review]
There are a lot of console errors with your addition:

validateDomNesting error:
Screen Shot 2021-03-24 at 8 35 08 PM


error when focused on subscribe to email input:
Screen Shot 2021-03-24 at 8 36 14 PM

error when visiting article page from main blog page:
Screen Shot 2021-03-24 at 8 35 18 PM

key error:
Screen Shot 2021-03-24 at 8 35 29 PM

**Comment by jamessspanggg** [comment]
> We are using styled components only on Slider but I think that it should not be a problem, because we have to redo the whole Slider for only that. 

I think there's no need to use `styled-components` as there are only two of your components that uses it, the rest uses a separate `scss` file. 

@angle943  @thesanjeevsharma thoughts?

**Comment by thesanjeevsharma** [comment]
Yeah, as the author mentioned he's gonna refactor it and remove styled-components this week. Should we wait until then? or we can allow this and later on merge the PR with the refactor?

IMO, it won't take much time to refactor so let's keep this open and wait for the refactor to be done first.

**Comment by IvanDanilov66** [comment]
All the errors are fixed @jamessspanggg can you please check now .

File: src/containers/TopNav/TopNavDesktopItems/index.tsx
Comment (angle943) [inline src/containers/TopNav/TopNavDesktopItems/index.tsx:65]: remove this comment

File: src/containers/TopNav/TopNavDesktopItems/index.tsx
Comment (angle943) [inline src/containers/TopNav/TopNavDesktopItems/index.tsx:67]: i don't think we need this comment

File: src/containers/Blog/Subscribe/index.tsx
Comment (angle943) [inline src/containers/Blog/Subscribe/index.tsx:20]: we have four nested `div` here, can we simplify this?

File: src/containers/Blog/Subscribe/Subscribe.scss
Comment (angle943) [inline src/containers/Blog/Subscribe/Subscribe.scss:105]: comment not needed

File: src/containers/Blog/Subscribe/Subscribe.scss
Comment (angle943) [inline src/containers/Blog/Subscribe/Subscribe.scss:40]: remove comments

File: src/containers/Blog/RecentlyPosted/index.tsx
Comment (angle943) [inline src/containers/Blog/RecentlyPosted/index.tsx:24]: this line is not needed, as you have a file-wide eslint-disable on top of this file

File: src/containers/Blog/RecentlyPosted/index.tsx
Comment (angle943) [inline src/containers/Blog/RecentlyPosted/index.tsx:21]: are these two `div` necessary? can we make them into 1?

File: src/containers/Blog/RecentlyPosted/Article.tsx
Comment (angle943) [inline src/containers/Blog/RecentlyPosted/Article.tsx:8]: what is this comment for?

File: src/containers/Blog/BlogHero/PopularArticles/PopularArticles.scss
Comment (angle943) [inline src/containers/Blog/BlogHero/PopularArticles/PopularArticles.scss:9]: let's remove these commented out styles

File: src/containers/Blog/BlogHero/PopularArticles/Article.tsx
Comment (angle943) [inline src/containers/Blog/BlogHero/PopularArticles/Article.tsx:26]: what's this?

File: src/containers/Blog/BlogHero/ActiveArticle/index.tsx
Comment (angle943) [inline src/containers/Blog/BlogHero/ActiveArticle/index.tsx:2]: we should remove these two eslint-disable

File: src/containers/Blog/Blog.scss
Comment (angle943) [inline src/containers/Blog/Blog.scss:51]: comment

File: src/containers/Blog/Blog.scss
Comment (angle943) [inline src/containers/Blog/Blog.scss:43]: comment

File: src/containers/Blog/Article/index.tsx
Comment (angle943) [inline src/containers/Blog/Article/index.tsx:75]: i think we can remove this

File: src/containers/Blog/Article/index.tsx
Comment (angle943) [inline src/containers/Blog/Article/index.tsx:11]: what is this comment for?

File: src/containers/Blog/Article/Article.scss
Comment (angle943) [inline src/containers/Blog/Article/Article.scss:155]: commented out style

File: src/containers/Blog/Article/Article.scss
Comment (angle943) [inline src/containers/Blog/Article/Article.scss:139]: commented out style here, plus width: 100% couple lines below

File: src/containers/Blog/Article/Article.scss
Comment (angle943) [inline src/containers/Blog/Article/Article.scss:103]: comment

File: src/containers/Blog/Article/Article.scss
Comment (angle943) [inline src/containers/Blog/Article/Article.scss:70]: comment

**Comment by angle943** [comment]
@IvanDanilov66 @jamessspanggg let's remove styled components before merging it in

**Comment by angle943** [comment]
@IvanDanilov66 are there any diffence between the contents inside `components/FormComponents` folder, and the one you created: `components/FormComponentsBlog`?

**Comment by jamessspanggg** [comment]
Updated the blog to be more responsive, as well as remove some unwanted styles as it seems like there are a lot of styles being copy and pasted that are unused.

Before:

https://user-images.githubusercontent.com/32864116/112849294-80ea2c00-90db-11eb-9899-41f8d7b0d603.mp4


After:

https://user-images.githubusercontent.com/32864116/112850314-77ad8f00-90dc-11eb-9c04-53fa83087a18.mp4

**Comment by angle943** [comment]
Closing this issue, as we have discussed about an alternative way to presenting blogs.

@IvanDanilov66 what is your account #?

**Comment by IvanDanilov66** [comment]
> Closing this issue, as we have discussed about an alternative way to presenting blogs.
> 
> @IvanDanilov66 what is your account #?

f1d85cecf85d7da38401dc58fabf1aa4294c881bb9d2c47642d1b460017f06cd

**Pull request closed without merging**
